src/Features/LanguageServer/ProtocolUnitTests/SemanticTokens/SemanticTokensRangeTests.cs

```diff
@@ -15,6 +15,8 @@
 using Xunit.Abstractions;
 using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
 
+#pragma warning disable format // We want to force explicit column spacing within the collection literals in this file, so we disable formatting.
```

Comment:
> We want to force explicit column spacing

1. This is never the case. Removing the extraneous spaces is desirable.
2. Suppression should not be applied to a file.

Reply:
There are 1000+ IDE0055 warnings in this file. Definitely feel free to create a follow-up PR to fix these up as appropriate and remove this suppression.